src/bin/vip-wp.js

```diff
@@ -111,6 +134,28 @@ const launchCommandAndGetStreams = async ( { guid, inputToken } ) => {
 	return { stdinStream, stdoutStream, socket };
 };
 
+const shutdownHandler = async ( terminateRunningCommand, guid ) => {
```

Comment:
can we make the signature more explicit here? The difference between `shutdownHandler( true, guid )` and `shutdownHandler( false, guid )` isn't explicit until you visit the function definition.

we can opt for: `shutdownHandler( guid, { exitProcess: true } )` or something similar

Reply:
Given your other comment about leaving the process.exit() I just removed that additional parameter from the method.  I had envisioned using the shutdownHandler in other cases and was trying to make it more generic.  If we need to use it in multiple places, we can revisit, but for now simplification is better.